src/core/core/events.js

```diff
@@ -1,13 +1,5 @@
 import EventEmitter from 'eventemitter2'
```

Comment:
I think we can remove this from `core` and just create the EventEmitter in `Router/index.js`. The only other place we use it is in `Modal/index.ks`, but it looks redundant to me so it can be removed